dist/gmusic.js

```diff
@@ -112,7 +112,7 @@ proto.volume = {
 
   // Get the current volume level.
   getVolume: function () {
-    return parseInt(this.volume._sliderEl.getAttribute('aria-valuenow'), 10);
+    return this.volume._sliderEl.__data__.value;
```

Comment:
It's not, I just got a little happy converting the stuff from getAttribute apparently..